src/main/resources/templates/form-fragments.html

```diff
@@ -23,6 +23,12 @@
 		</label>
 		<span th:errors="${beverage.name}" class="error"></span>
 	</div>
+	<div class="form-group md-4">
+	<label class="font-weight-bold">Beverage Description
+		<input th:field="${beverage.description}" class="form-control">
```

Comment:
You might consider making this a text-area input like we did in the recipe class. It would help with seeing longer descriptions!

Reply:
Great suggestion, thanks!